modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePostgreSQLTest.java

```diff
@@ -18,22 +19,36 @@
 public class SimplePostgreSQLTest {
 
     @Rule
-    public PostgreSQLContainer postgres = new PostgreSQLContainer();
+    public PostgreSQLContainer postgres = (PostgreSQLContainer) new PostgreSQLContainer().withCommand("postgres -c max_connections=42");
 
     @Test
     public void testSimple() throws SQLException {
+        ResultSet resultSet = performQuery("SELECT 1");
+        int resultSetInt = resultSet.getInt(1);
+        assertEquals("A basic SELECT query succeeds", 1, resultSetInt);
+    }
+
+    @Test
+    public void testCommandOverride() throws SQLException {
+        ResultSet resultSet = performQuery("SELECT current_setting('max_connections')");
+        String result = resultSet.getString(1);
+        assertEquals("max_connections should be overriden", "42", result);
+    }
+
+    @NonNull
```

Comment:
I'm not 100% sure, but we might want to get rid of those annotation, I have to check in with the others.